roles/pulibrary.figgy/tasks/main.yml

```diff
@@ -126,4 +126,6 @@
 - name: Compile simple-tiles from source
   shell: curl -L {{ simple_tiles_url }} | tar -xz && cd simple-tiles-{{simple_tiles_version}} && ./configure && make && make install
   args:
-    creates: /usr/local/lib/libsimple-tiles.so
\ No newline at end of file
+    creates: /usr/local/lib/libsimple-tiles.so
+- name: Copy tika config
+  copy: src=files/tika_config.xml dest=/opt/tika/tika_config.xml
```

Comment:
I don't think this'll work because `/opt/tika` doesn't exist and isn't owned by `deploy`